docs/Reply.md

```diff
@@ -70,6 +70,9 @@ reply
   .header('Content-Type', 'application/x-protobuf')
   .serializer(protoBuf.serialize)
 ```
+
+Note that if a string or buffer is passed to `reply.send` it is expected to already be serialised and skip the serialization step.
```

Comment:
Use `serialized` or `serialised` but it should be consistent. We already said `serialized` in the docs.